images/kubekins-e2e/Dockerfile

```diff
@@ -58,6 +58,9 @@ ENV IMAGE=${IMAGE_ARG}
 # preinstall this for kops tests xref kubetest prepareAws(...)
 RUN pip install awscli
 
+# preinstall graphviz package for graphing profiles
+RUN apt-get install -y graphviz
```

Comment:
This should not be another step.

Reply:
Could you clarify? Do you mean I should add it to some already existing `RUN` command above?

Reply:
This image has an unfortunately long build and number of layers, I thought we still had another `apt-get` call here.

This probably make no difference, I withdraw. I do wonder if this should be in the image though.